actions/types.ts

```diff
@@ -182,6 +182,12 @@ export interface InvestigatorCampaignNoteCount {
   custom?: boolean;
 }
 
+export interface ChaosBagResults {
+  currentToken?: ChaosTokenType;
```

Comment:
How heavy is the divide between current + drawn.  Is there a world where current tokens is just the 'first' drawn token?

Reply:
Not really that heavy, it could easily be the first from the drawn tokens.